src/main/java/org/tikv/common/operation/iterator/ConcreteScanIterator.java

```diff
@@ -75,9 +75,9 @@ TiRegion loadCurrentRegionToCache() throws GrpcException {
     TiRegion region;
     try (RegionStoreClient client = builder.build(startKey)) {
       client.setTimeout(conf.getScanTimeout());
-      region = client.getRegion();
       BackOffer backOffer = ConcreteBackOffer.newScannerNextMaxBackOff();
       currentCache = client.scan(backOffer, startKey, version);
+      region = client.getRegion();
```

Comment:
We might add some comments about this update since the `scan` has some implicit side effects. In addition, this issue might occur in https://github.com/tikv/client-java/blob/9ce52d34017e2caaba1a74335e454a87ebeed2af/src/main/java/org/tikv/common/operation/iterator/RawScanIterator.java#L49, the raw scan of client-java. We might add some mock tests for this interface, at least a **TODO** should be added.

Reply:
I have add comment and fix it in RawScanIterator. I think test it is not easy, so a TODO is added.